antarest/core/utils/utils.py

```diff
@@ -80,6 +89,30 @@ def new_redis_instance(config: RedisConfig) -> redis.Redis:
     return redis.Redis(host=config.host, port=config.port, db=0)
 
 
+def create_new_empty_study(
```

Comment:
this should go to antarest/study/storage/utils.py

Reply:
we don't want to handle RawStudy, just the optional version and a path to extract the new study to.
the rest (updating RawStudy, and the files from RawStudy should be handled elsewhere).